application/classes/Ushahidi/Core.php

```diff
@@ -542,6 +542,34 @@ public static function init()
 		$di->setter['Ushahidi_Listener_PostSetListener']['setRepo'] =
 			$di->lazyGet('repository.notification.queue');
 
+		// Login rate limiter
+		$di->set('ratelimiter.login', $di->lazyNew('BehEh\Flaps\Flap'));
+
+		$di->params['BehEh\Flaps\Flap'] = [
+			'storage' => $di->lazyGet('ratelimiter.storage'),
+			'name' => 'login'
+		];
+
+		// Rate limiter storage
+		$di->set('ratelimiter.storage', $di->lazyNew('BehEh\Flaps\Storage\DoctrineCacheAdapter'));
```

Comment:
For prod:
- Maybe use something like the FilesystemCache Adapter or something that'll work without extra tooling by default 
- We did have field for storing failed login counts etc in the DB.. but maybe that doesn't make sense anymore?
- Should ship an option for using memcache since we can deploy that easily to ushahidi.io